### PR TITLE
[W08H02] Adds a Test if epuals works correctly

### PR DESCRIPTION
**Pull request by Benn1x**
The Change is in the Test for TaskPool and checks if equales work right it is simply the test from artemis but for the sake of completeness i would add them here to

### CODE REVIEW

**Review by HendrikHuebner — APPROVED** [review]
LGTM

File: w08h02/test/pgdp/pools/TaskPoolBehaviorTest.java [hunk 1]
```diff
@@ -51,6 +51,21 @@ void artemisExample4() {
         assertEquals(task, pool.getByValue(1, func), "Querying a Task with getByValue which was added before should yield that task.");
     }
 
+    @Test
+    @DisplayName("Check if Compare works right")
+    void compareCheck(){
```
Comment (nilsreichardt) [inline w08h02/test/pgdp/pools/TaskPoolBehaviorTest.java:56]:
```suggestion
    void compareCheck() {
```

Nit: Just code styling

**Review by nilsreichardt — APPROVED** [review]
LGTM 🎉

**Review by MaximilianAnzinger — COMMENTED** [review]
🐧

**Pull request merged**